src/core/js/models/configModel.js

```diff
@@ -16,13 +16,35 @@ export default class ConfigModel extends LockingModel {
     };
   }
 
+  /**
+   * Parse the incoming search queries for certain parameter values to use as defaults
+   */
+  setValuesFromURLParams() {
+    const paramMappings = {
+      dir: '_defaultDirection',
+      lang: '_defaultLanguage'
+    };
+
+    const params = new URLSearchParams(window.location.search);
+
+    for (const [key, value] of Object.entries(paramMappings)) {
+      const passedVal = params.get(key);
+
+      if (!passedVal) continue;
+
+      this.set(value, passedVal);
+    }
+  }
```

Comment:
This works just as well:
```suggestion
    Object.entries(paramMappings).forEach(([key, value]) => {
      const passedVal = params.get(key);
      if (!passedVal) return;
      this.set(value, passedVal);
    });
  }
```